install/README.md

```diff
@@ -15,6 +15,11 @@ In this quickstart, we will create a Kubernetes cluster, and populate it with th
    1. [Installing Minikube](#installing-minikube)
    1. [Creating an agones profile](#creating-an-agones-profile)
    1. [Starting Minikube](#starting-minikube)
+1. [Setting up an Amazon Web Services EKS cluster](#setting-up-an-amazon-web-services-eks-cluster)
+    1. [Create EKS Instance](#create-eks-instance)
+    1. [Ensure VPC CNI 1.2 is Running](#ensure-vpc-cni-12-is-running)
+    1. [Follow Normal Instructions to Install](#follow-normal-instructions-to-install)
+    1. [Add HostPort to fleet or gameserver instructions](#add-hostport-to-fleet-or-gameserver-instructions)
```

Comment:
I believe this should go too? :smile: 

Also, please rebase down to a single commit - and once this header is gone, happy to approve and merge.

Reply:
👍